app/Services/UserService.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\User;
+
+class UserService
+{
+    public function registerUser($userData)
+    {
+
+        $user = User::create([
+            'name' => $userData['name'],
+            'email' => $userData['email'],
+            'password' => bcrypt($userData['password']),
+        ]);
+
```

Comment:
instead of passing all the data in an array you can do it like this  return User::query()->create($data);